scripts/lint.sh

```diff
@@ -25,7 +26,12 @@ else
   TARGET="${1}"
 fi
 
-add_license_headers -check
+ADD_LICENSE=${ADD_LICENSE:-true}
+if [ "$ADD_LICENSE" ]; then
+  add_license_headers -check
+fi
```

Comment:
The `find` is probably wrong when running from anything outside the repo root. We also don't really need to add licenses since it should have already been done by the script at `<root>/scripts/lint.sh`